---
layout: archive
title: "Publications (Selected) "
permalink: /research/
author_profile: true
---

<h2> Recently Published Papers </h2>
Trainees are underlined, and the last/corresponding authors are denoted by *. The impact factor (when provided) is 2023. A full list of publications can be found on my <a href="https://scholar.google.ca/citations?user=5QsFxcAAAAAJ&hl=en"> Google Scholar </a> page.


<h3> Single Cell Genomics</h3> 
<ol>
	<li>	<u>Kailun Bai</u>, Belaid Moa, Xiaojian Shao, <b>Xuekui Zhang</b>*. scSorterDL: A Deep Neural Network-Enhanced Ensemble LDAs for Single Cell Classifications. 2025, <em>Briefings in Bioinformatics</em>  <a href="https://doi.org/10.1093/bib/bbaf446"> Link to Paper </a>  </li>
	<li>	Yao Dong, Chen Chen, Jin Shi, <u>Yushan Hu</u>, <u>Xiaowen Cao</u>, Yongfeng Dong and <b>Xuekui Zhang</b>*. MOH: a novel multilayer multi-omics heterogeneous graph for single-cell clustering. 2025, <em>IEEE Journal of Biomedical and Health Informatics</em>, 2025, <a href="https://doi.org/10.1109/JBHI.2025.3589464"> Link to Paper </a>  </li>  
	<li>	<u>Chang Bi</u>, <u>Kailun Bai</u>, <b>Xuekui Zhang</b>*. HiCat: A Semi-Supervised Approach for Cell Type Annotation. 2025,  <em>Briefings in Bioinformatics</em>,  <a href="https://doi.org/10.1093/bib/bbaf428"> Link to Paper </a>   </li>
	<li>	<u>Kailun Bai</u>, Belaid Moa, Xiaojian Shao, and <b>Xuekui Zhang</b>*.  PCLDA: An interpretable cell annotation tool for single-cell RNA-sequencing data based on simple statistical methods. 2025, <em>Computational and Structural Biotechnology Journal</em>, <a href="https://doi.org/10.1016/j.csbj.2025.07.019"> Link to Paper </a>  </li>
	<li>	Yao Dong, Jiaxue Zhang, Jin Shi, <u>Yushan Hu</u>, <u>Xiaowen Cao</u>, Yongfeng Dong, and <b>Xuekui Zhang</b>*. scAGCI: an anchor graph-based method for cell clustering from integrated scRNA-seq and scATAC-seq data. 2025, <em>Briefings in Bioinformatics</em>,  <a href="https://doi.org/10.1093/bib/bbaf244"> Link to Paper </a> and <a href="https://github.com/hebutdy/scAGCI"> GitHub </a>  </li>
	<li>	<u>Yushan Hu</u>, Xiaojian Shao, Li Xing, Xuan Li, Geoffrey M. Nonis, Graeme J. Koelwyn*, <b>Xuekui Zhang</b>*, Dong D. Sin*. Single Cell Sequencing of Lung Macrophages and Monocytes Reveals Novel Therapeutic Targets in COPD. 2023, <em>Cells</em>   <a href="https://doi.org/10.3390/cells12242771"> Link to Paper </a>  </li>
	<li>	<u>Xiangling Ji X</u>, <u>Danielle Tsao</u>, <u>Kailun Bai K</u>, Min Tsao, Li Xing, and <b> Xuekui Zhang</b>*. scAnnotate: an automated cell type annotation tool for single-cell RNA-sequencing data. 2023, <em>Bioinformatics Advance</em>,  <a href="https://doi.org/10.1093/bioadv/vbad030"> Link to Paper </a> and <a href="https://cran.r-project.org/package=scAnnotate"> R package </a>  </li>
</ol>	


<h3> Statistics, Bioinformatics, and Biomedicine </h3> 
<ol>	
<Li>  	<b>Xuekui Zhang</b>*, Li Xing*， <u>Jing Zhang</u>, <u>Kim Soojeong</u> . Beyond Conventional P-Values: Addressing Statistical Challenges in Big Data. 2025, <em>Communications in Statistics - Simulation and Computation</em>  <a href="https://doi.org/10.1080/03610918.2025.2593940"> Link to Paper </a>  </li>
<Li>  	<u>Xiaowen Cao</u>, <u>Shuai You</u>, Grace Y. Yi, <b>Xuekui Zhang</b>*, Li Xing*. Concurrent Prediction of Multiple Survival Outcomes with a Refined Stacking Algorithm. 2025, <em>Journal of Computational and Graphical Statistics</em>  <a href="https://doi.org/10.1080/10618600.2025.2561900"> Link to Paper </a>  </li>
<Li>    Bie Y, <u>You S</u>, Li X, <b>Zhang X</b>* and Lu T*, Essential Number of Principal Components and Nearly Training-Free Model for Spectral Analysis. 2024, <em>IEEE Transactions on Pattern Analysis and Machine Intelligence</em>, (Impact factor 20.8) <a href="https://doi.org/10.1109/TPAMI.2024.3436860"> Link to Paper </a> </li>
<Li>    <u>Ning H</u>, Boyes I, Numanagić I, Rott M*, Xing L*, <b>Zhang X</b>*. Diagnostics of viral infections using high-throughput genome sequencing data. 2024, <em>Briefings in Bioinformatics</em>   <a href="https://doi.org/10.1093/bib/bbae501"> Link to Paper </a> and <a href="https://cran.r-project.org/package=iimi"> R package </a>  </li>
<li>	Luo W, Zhu Y, <b>Zhang X</b>, Lin L. A, Semi-parametric Estimation of Personalized Dose-response Function Using Instrumental Variables. 2024, <em>Journal of Machine Learning Research </em>, <a href="https://www.jmlr.org/papers/v25/21-1181.html"> Link to Paper </a> </li>
<li>	<b>Zhang X</b>, <u>Huang X</u>, and Xing L. ADSP: An Adaptive Sample Pooling Strategy for Diagnostic Testing. 2023, <em>Journal of Biomedical Informatics</em>,  <a href="https://www.sciencedirect.com/science/article/pii/S1532046423002228"> Link to Paper </a>   </li>	
<li>  Cheng X, Liu Y, Wang J, Robertson G, <b>Zhang X</b>, Jones S, and Taubert S.  cSurvival: a web resource for biomarker interactions in cancer outcomes and in cell lines. 2022, <em> Briefings in Bioinformatics</em> (impact factor 9.5) <a href="10.1093/bib/bbac090"> Link to Paper </a>  </li>
<li>   Xing L, <b>Zhang X</b>, Burstyn I， Gustafson P. On logistic Box–Cox regression for flexibly estimating the shape and strength of exposure‐disease relationships. 2021，<em>Canadian Journal of Statistics</em>  <a href="https://onlinelibrary.wiley.com/doi/10.1002/cjs.11587"> Link to Paper </a>,  （<a href="https://ssc.ca/en/awards?field_award_year_value%5Bvalue%5D%5Byear%5D=&tid=1320"> <strong> Statistics Society of Canada 2022 CJS Award </strong> </a> ）</li>
<li> Xing L, Lesperance ML, <b>Zhang X</b>*. Simultaneous prediction of multiple outcomes using revised stacking algorithms. 2020， <em>Bioinformatics</em> <a href="https://doi.org/10.1093/bioinformatics/btz531"> Link to Paper </a> and 
<a href="https://cran.r-project.org/web/packages/MTPS/index.html"> Software package </a> </li>
</ol>	


<h3> Environmental Research </h3> 
<ol> 
<li>	<u>Bi C</u>, Pan Y*, <b>Zhang X</b>*. Predicting Paralytic Shellfish Poisoning on the West Coast of Canada. 2025 <em>Journal of Hazardous Materials</em> (impact factor 11.3) <a href="https://doi.org/10.1016/j.jhazmat.2025.140459"> Link to Paper </a>  </li>
<li>	<u>Liu Z</u>, <u>You S</u>, Xing L, Durand G, Moccia P, Mercier V, Pan Y*, <b>Zhang X</b>*. OPMS - a web-based ocean pollution monitoring system. 2025 <em>Marine Pollution Bulletin</em> (impact factor 5.8) <a href="https://doi.org/10.1016/j.marpolbul.2024.117510"> Link to Paper </a>  </li>
<li>	<u>You S</u>, Xing L, Lesperance M, Pan Y*, <b>Zhang X</b>*. Temporal and Spatial Variation Patterns of Domoic Acid along Canada’s Coast. 2024 <em>Science of the Total Environment</em>  (Impact Factor 9.8) <a href="https://doi.org/10.1016/j.scitotenv.2024.172817"> Link to Paper </a>  </li>
<li>	<u>You S</u>, Xing L, Lesperance M, Pan Y*, <b>Zhang X</b>*. Longitudinal Study of Paralytic Shellfish Toxins along Canada’s Coast. 2024 <em>Environmental Research</em> (Impact Factor 8.3) <a href="https://doi.org/10.1016/j.envres.2024.118944"> Link to Paper </a>  </li>
<li>	<u>You S</u>, <u>Huang X</u>, Xing L, Lesperance M, LeBlanc C, Moccia P, Mercier V, Shao X, Pan Y*, <b>Zhang X</b>*. 2023,  Dynamics of Fecal Coliform Bacteria along Canada’s Coast. <em>Marine Pollution Bulletin</em> (impact factor 5.8) <a href="https://doi.org/10.1016/j.marpolbul.2023.114712"> Link to Paper </a>  </li>
</ol>	


<h3> Design of adaptive clinical trials </h3> 
<ol>
	<li>	Cong Chen, Mo Huang, <b>Xuekui Zhang</b>. Adaptive Phase 2/3 Design with Dose Optimization. 2025 <em>Contemporary Clinical Trials </em> <a href="https://doi.org/10.1016/j.cct.2025.108048"> Link to Paper </a> </li> 
	<li>	Cong Chen, Mo Huang, <b>Xuekui Zhang</b>. Adjustment for Inconsistency in Adaptive Phase 2/3 Designs with Dose Optimization. 2025 <em>Pharmaceutical Statistics </em>  <a href="https://doi.org/10.1002/pst.70031"> Link to Paper </a> </li> 
	<li>	Cong Chen, Linda Sun, <b>Xuekui Zhang</b>.  A promising biomarker adaptive Phase 2/3 design – explained and expanded. 2023, <em>Contemporary Clinical Trials Communications</em>  <a href="https://doi.org/10.1016/j.conctc.2023.101229"> Link to Paper </a> </li> 
<li>	Cong Chen, and <b>Xuekui Zhang</b>. From bench to bedside, 2-in-1 design expedites phase 2/3 oncology drug development. 2023, <em>Frontiers in Oncology</em>,  <a href="https://doi.org/10.3389/fonc.2023.1251672"> Link to Paper </a> </li> 
<li>	<b>Xuekui Zhang</b>, <u>Haijun Jia</u>, Li Xing, and Cong Chen. Application of group sequential methods to the 2-in-1 design and its extensions for interim monitoring. 2023, <em>Statistics in Biopharmaceutical Research</em>,  <a href="https://www.tandfonline.com/eprint/MBTZAZGJUFAHDMW3M7RV/full?target=10.1080/19466315.2023.2197402"> Link to Paper </a> </li>
<li>    <u>Yitao Lu</u>, Julie Zhou, Li Xing, <b>Xuekui Zhang</b>*. The Optimal Design of Clinical Trials with Potential Biomarker Effects, A Novel Computational Approach. 2021， <em> Statistics in Medicine</em>  <a href="https://doi.org/10.1002/sim.8868"> Link to Paper </a> and 
<a href="https://cran.r-project.org/web/packages/DesignCTPB/index.html"> Software package </a>   </li>  <!-- <em>(<small> The first author is an intern undergraduate student in my lab, who lead programming and 1st draft preparation of this project.</small>)</em>  -->
	
</ol>	

<h3> Economics </h3> 
<ol>
<li> <b>Zhang X</b>, <u>Huang Y </u>, Xu K, Xing L. Novel Modelling Strategies for High-frequency Stock Trading Data. 2023，<em> Financial Innovation</em> (impact factor 8.4) <a href="https://doi.org/10.1186/s40854-022-00431-9"> Link to Paper </a> and <a href="https://github.com/ubcxzhang/Stock-Price-Prediction"> GitHub code </a>  </li>	
<li>Xu K, Zheng X, Xing L, <b>Zhang X</b>. Stock Market Openness and Market Quality: Evidence from the Shanghai-Hong Kong Stock Connect Program. 2020， <em>Journal of Financial Research</em>	<a href="https://doi.org/10.1111/jfir.12210"> Link to Paper </a> </li>
</ol>	

<h3> COVID-19 Research </h3> 
<ol>
<li>   <u>Li M</u>, Esfahani F, Xing L, <b>Zhang X</b>*.  Predicting the daily counts of COVID-19 infection using temporal convolutional networks. 2023, <em>Journal of Global Health</em> (impact factor 7.2), <a href="https://doi.org/10.7189/jogh.13.03029"> Link to Paper </a>  <em>(<small> The first author is an intern high-school student in my lab, who lead programming and 1st draft preparation of this project.</small>)</em> </li>
<li>   <u>Guo Y </u>, Esfahani F, Shao X, Srinivasan V, Thomo A, Xing L, <b>Zhang X</b>*. Integrative COVID-19 Biological Network Inference with Probabilistic Core Decomposition. 2021，<em> Briefings in Bioinformatics</em> (impact factor 9.5) <a href="https://academic.oup.com/bib/article/23/1/bbab455/6425808"> Link to Paper </a>  </li>		
<li>   <u>Huang X </u>, Shao X, Xing L, <u>Hu Y</u>, Sin DD*, <b>Zhang X</b>*. The impact of lockdown timing on COVID-19 transmission across US counties. 2021， <em> EClinicalMedicine</em> (a journal published by the Lancet, impact factor 15.1) <a href="https://www.thelancet.com/journals/eclinm/article/PIIS2589-5370(21)00315-1/fulltext"> Link to Paper </a>  </li>
</ol>	


<h2>Software </h2>
<p> <strong>scAnnotate</strong>: An Automated Cell Type Annotation Tool for Single-Cell RNA-Sequencing Data.   <a href="https://cran.r-project.org/web/packages/scAnnotate/index.html"> CRAN</a> </p>
<p> <strong>DesignCTPB</strong>: Design Clinical Trials with Potential Biomarker Effect.   <a href="https://cran.r-project.org/web/packages/DesignCTPB/"> CRAN</a> </p>
<p> <strong>MTPS</strong>: Multi-Task Prediction using Stacking Algorithms.   <a href="https://cran.r-project.org/web/packages/MTPS/"> CRAN</a> </p>
<p> <strong>GWASbyCluster</strong>: Identifying disease-associated significant SNPs using clustering approach.   <a href="https://cran.r-project.org/web/packages/GWASbyCluster"> CRAN </a> </p>
<p> <strong>PICS</strong>: Probabilistic inference of ChIP-Seq using an empirical Bayes mixture model approach.   <a href="https://bioconductor.org/packages/release/bioc/html/PICS.html"> Bioconductor </a> </p>
<p> <strong>PING</strong>: Probabilistic inference for Nucleosome Positioning with MNase-based or Sonicated Short-read Data.   <a href="https://bioconductor.org/packages/release/bioc/html/PING.html"> Bioconductor </a> </p><br>

<!--
<h2>Funded Research Projects (Selected) </h2>
<p>  <a href="https://www.genomecanada.ca/en/programs/translation/funding-opportunities/genomic-applications-partnership-program"> CIHR project grant </a> (2020-2025, <em><u>$378,676</u></em>) Developing a Patient-Specific Surgical Decision-Assistance Tool to Improve Reverse
Shoulder Replacement Outcomes  (Role: one of three Principal Investigators)<br>
<p>  <a href="https://www.genomecanada.ca/en/programs/translation/funding-opportunities/genomic-applications-partnership-program"> Genome Canada GAPP </a> (2020-2024, <em><u>$6,292,438</u></em>) Next Generation Diagnostics for Clean Grapevines in Canada  (Role: Academic Co-Leader)<br>
<p>  <a href="https://www.chairs-chaires.gc.ca/home-accueil-eng.aspx#"> Canada Research Chair in Biostatistics and Bioinformatics (Tier 2) </a> (2017-2022, <em><u>$580,000</u></em>)  (Role: Principal Investigator) <br>
<p>  <a href="https://www.nserc-crsng.gc.ca/professors-professeurs/grants-subs/dgigp-psigp_eng.asp"> NSERC Discovery Grant </a> (2017-2022, <em><u>$95,000</u></em>)  Integrated analysis of time-course multi-type Next Generation Sequencing data and biomarker discovery using multiple studies (Role: Principal Investigator) <br>
<p>  <a href="https://cihr-irsc.gc.ca/e/49051.html"> CIHR Project Grant	 </a> (2020-2023, <em><u>$443,700</u></em>)  Childhood Obesity Management Using Innovative Digital Technology (Role: Co-Applicant)<br>	
<p>  <a href="https://www.computecanada.ca/research-portal/accessing-resources/resource-allocation-competitions/"> Compute Canada Resource allocation competition </a> (2019-current)  Development of Novel Statistical Methods and software in Complex Genomics Data (Role: Principal Investigator)<br>	 -->
